src/websocket_endpoint.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "websocketpp/client.hpp"
+#include "websocketpp/config/asio_no_tls_client.hpp"
+#include "websocketpp/extensions/permessage_deflate/enabled.hpp"
+#include <functional>
+#include <thread>
+#include <memory>
+namespace sensr
+{
+    typedef websocketpp::client<websocketpp::config::asio_client> websocketpp_client;
```

Comment:
Why does this use `typedef` unlike `MsgReceiver`? Also, this can be moved into `WebSocketEndPoint`.

Reply:
I'm not the kind of "format popo", but I hope all the SDK code has a consistent format, and even better if it follows google convention. 
For example, The tab space here is 4 but some of them use 2, as well as inconsistent use of `#pragma once` and `#ifndef`.

Reply:
Sorry, the code here was copied from websocket tutorial. I've updated indentation and #pragram stuff.